bluebottle/bluebottle_dashboard/static/jet/js/src/features/sidebar/main.js

```diff
@@ -3,7 +3,6 @@ var SideBarApplicationPinning = require('./application-pinning');
 var SideBarBookmarks = require('./bookmarks');
 var SideBarPopup = require('./popup');
 
-require('perfect-scrollbar/jquery')($);
```

Comment:
Is it right to remove this?

Reply:
Fixed by installing the default package with the same version instead of the one from github.